scapy/utils.py

```diff
@@ -852,7 +852,12 @@ def read_all(self,count=-1):
         res=[]
         while count != 0:
             count -= 1
-            p = self.read_packet()
+            try:
+                p = self.read_packet()
+            except:
+                if conf.debug_dissector:
```

Comment:
That's not how `conf.debug_dissector` is used for. It is used to let exceptions happen when dissecting a packet, rather than using `Raw()` instead.

Why do you need this change?

Reply:
Was used while debugging. Removed